Data lineage doc review (#738)
astronomer/docs: astro/data-lineage.md

## Overview

The **Lineage** tab in the Cloud UI can help you troubleshoot issues with your data pipelines and understand the movement of data across your organization.

From the **Lineage** tab on Astro, you can access the following four pages:

- **Explore**: A real-time overview of all **jobs** that emit data lineage across your organization. A job can be an Airflow task or any other process configured to emit lineage data to Astronomer, such as a Spark job.
- **Issues**: A view of potential issues or statistical inconsistencies related to your jobs or datasets.
- **Lineage**: A graph view that visualizes data lineage.
- **Integrations**: A view of your current data lineage integrations. 

You can use these pages to diagnose issues that may be difficult to troubleshoot in other environments. For example, if an Airflow task failed because a database schema changed, you can use the **Lineage** page on Astro to determine which job caused the change and which downstream tasks failed as a result.

> All members of your organization can view the **Lineage** page. For more information on data lineage and related concepts, see [Data Lineage Concepts](data-lineage-concepts.md).

Reviewer: ```suggestion
All members of your Organization can view the **Lineage** page. For more information on data lineage and related concepts, see [Data Lineage Concepts](data-lineage-concepts.md).
```
The reason we capitalize "Organization" in Astro is because it is an RBAC entity that has specific connotations. We are in a similar problem with our usage of "Deployment", where we're not quite sure how to make it clear that we're talking about an Astro Deployment without capitalizing or continually referencing Astro. Would love your help with a solution to documenting/ formatting these system components, but for now let's keep Organization capitalized.

:::

## View the Lineage Graph for a Data Pipeline

You can use the search field at the top of the Cloud UI to view the lineage graph for one of your data pipelines, search for a DAG, task, or dataset. You can also search for jobs from other tools with lineage integrations, including dbt or Spark.

![Example query in the lineage search bar](/img/docs/lineage-search.png)

The search results include the namespace that emitted the matching event. When an Astro Deployment emits the lineage event, the namespace matches the Deployment namespace shown in the **Deployments** page of the Cloud UI. Clicking a search result opens the **Lineage** page and shows the lineage graph for the selected job or dataset. You can also access the lineage graph for a recent job run in the **Explore** page below **Most Recent Runs**.

The **Lineage** page shows lineage data only for the most recent run of a given data pipeline. To explore lineage data from previous runs, see [Compare Lineage Graphs from Previous Runs](data-lineage.md#compare-lineage-graphs-from-previous-runs).

:::info

> By default, when you access the **Lineage** page from the left menu, the last lineage graph you viewed is displayed. If you go directly to the **Lineage** page without viewing a lineage graph, no lineage graph data is displayed. If this happens, access a job run using the search bar or use the **Explore** page to populate the **Lineage** page with data.

Reviewer 2: ```suggestion
By default, when you access the **Lineage** page from the left menu, the last lineage graph you viewed is displayed. If you go directly to the **Lineage** page without viewing a lineage graph, no lineage graph data is displayed. If this happens, you can access a job run using the search bar or use the **Explore** page to populate the **Lineage** page with data.
```

:::

:::info

> A Lineage graph with a single node indicates that the job you selected didn't emit any information about input or output datasets. Typically, this occurs when an Airflow task isn't using a [supported Airflow operator](data-lineage-support-and-compatibility.md). You can still view the duration of this job over time.

Reviewer: ```suggestion
A lineage graph with a single node indicates that the job you selected didn't emit any information about input or output datasets. Typically, this occurs when an Airflow task isn't using a [supported Airflow operator](data-lineage-support-and-compatibility.md). You can still view the duration of this job over time.
```

![Quality tab example](/img/docs/quality-tab.png)

These charts display data for the past 20 job runs.

Use the **Quality** tab to detect unexpected changes or statistical variance in your dataset that could indicate a problem. The following topics explain each available chart in the tab.

#### Rows

> The Rows chart shows the total number of rows in the dataset over time. A significant change in rows can occur naturally (for example: when a rapid increase of customer orders occurs during the holiday season). However, it can also indicate an error in an upstream job, especially if it is sudden or unexpected.

Reviewer 2: ```suggestion
The Rows chart shows the total number of rows in the dataset over time. A significant change in rows can occur naturally. For example, a rapid increase of customer orders occurs during the holiday season. However, it can also indicate an error in an upstream job, especially if it is sudden or unexpected.
```

#### Bytes

> The Bytes chart shows the total size of the dataset over time. A sudden increase in dataset size usually indicates something has changed in the definition of the data. For example, a new column might have been added to your table containing the description of an order, where before it contained only part numbers and quantities.

Reviewer 2: ```suggestion
The Bytes chart shows the total size of the dataset over time. A sudden increase in dataset size usually indicates that something has changed in the definition of the data. For example, a new column might have been added to your table containing the description of an order, where before it contained only part numbers and quantities.
```

#### Quality Metrics

The **Quality Metrics** chart shows the pass or fail status of quality assertions in a Great Expectations suite.
To see details on the assertions that have passed or failed, hover over a point on the chart.

#### Distinct Count (Column-level)

The **Distinct Count** chart shows the total number of distinct values for a given column.

> A distinct count can sometimes grow unexpectedly. For example, a successful campaign suddenly creates a set of new `customer_id` values in an order table. However, it can also suggest an underlying problem if, for example, a `menu_item_id` field shows that thousands of new menu items have been added overnight.

Reviewer 2: ```suggestion
Distinct count can sometimes grow unexpectedly. For example, a successful marketing campaign might suddenly create a set of new `customer_id` values in an order table. However, it can also suggest an underlying problem if, for example, a `menu_item_id` field shows that thousands of new menu items have been added overnight.
```
